### PR TITLE
Lpd 15146 v2

### PR DESCRIPTION
**Pull request by manuelRives**

### CODE REVIEW

**Comment by manuelRives** [comment]
ci:test:headless

**Comment by manuelRives** [comment]
ci:test:headless

**Comment by jkappler** [comment]
ci:test:headless

**Comment by manuelRives** [comment]
ci:report:18133170

File: modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/BaseDocumentResourceTestCase.java [hunk 1]
```diff
@@ -4430,35 +4438,38 @@ else if (operator.equals("startswith")) {
 			return sb.toString();
 		}
 
-		if (entityFieldName.equals("datePublished")) {
-			if (operator.equals("between")) {
-				Date date = document.getDatePublished();
+		if (FeatureFlagManagerUtil.isEnabled(
+			testCompany.getCompanyId(), "LPD-10701")) {
```
Comment (robertoDiaz) [inline modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/BaseDocumentResourceTestCase.java:4442]: Could be an AND in order to save space

File: modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/BaseDocumentResourceTestCase.java [hunk 2]
```diff
@@ -3765,15 +3769,19 @@ protected boolean equals(Document document1, Document document2) {
 				continue;
 			}
 
-			if (Objects.equals("datePublished", additionalAssertFieldName)) {
-				if (!Objects.deepEquals(
+			if (FeatureFlagManagerUtil.isEnabled(
+				testCompany.getCompanyId(), "LPD-10701")) {
+				if (Objects.equals(
```
Comment (robertoDiaz) [inline modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/BaseDocumentResourceTestCase.java:3774]: could be an AND

**Review by robertoDiaz — CHANGES_REQUESTED** [review]
2 small changes requested

**Pull request closed without merging**